riscv/src/reachability.rs

```diff
@@ -10,14 +10,13 @@ pub fn filter_reachable_from(
 ) {
     let replacements = extract_replacements(statements);
     let label_offsets = extract_label_offsets(statements);
-    let mut queued_labels: BTreeSet<&str> = vec![label].into_iter().collect();
-    let mut referenced_labels: BTreeSet<&str> = vec![label].into_iter().collect();
+    let mut queued_labels = BTreeSet::from([label]);
+    let mut referenced_labels = BTreeSet::from([label]);
```

Comment:
This seems unused?

Reply:
It is used. Please expand the code until the end of the function.